x-pack/plugins/monitoring/public/lib/ajax_error_handler.tsx

```diff
@@ -11,7 +11,7 @@ import { EuiButton, EuiSpacer, EuiText } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
```

Comment:
I think this file is not needed anymore and it's a leftover from the react migration 😱 
I'm going to check it now, and create a PR if it's not needed anymore

Reply:
Duplicated comment 🙃